### PR TITLE
Defining all the trainings that need to be compared

### PR DESCRIPTION
**Pull request by czifrushanna**
Working on #30 

The code for all the training is ready.
Still to come: comparing and visualising the results.

### CODE REVIEW

**Comment by zeef69** [comment]
Probléma 1
`Classifier class` -->`test_step(self, batch, batch_idx)` metódus:
A visszatérési érték return accuracy kell legyen, nem?

**Comment by zeef69** [comment]
Probléma 2
Ebben az esetben a checkpointnál az újrabetöltéskor miért a VICReg-ét töltjük be, hiszen nem azon volt tanítva.
Fejezet: **Without SSL, but pretrained backbone with frozen weights** 

`# Before running change ckpt_with_classifier_path to the correct one`
`ckpt_with_classifier_path = "models/best_model_with_pretrained_frozen.ckpt"`
`ssl_model = VICReg()`
`model_pretrained_frozen = Classifier(ssl_model.backbone, False, num_classes)`
`model_pretrained_frozen.load_state_dict(torch.load(ckpt_with_classifier_path, map_location=torch.device('cpu'))['state_dict...`

**Comment by czifrushanna** [comment]
Probléma 1: Igazából végül nem is használjuk most a test_step-et, csak a predict_step-et jelen pillanatban. Egyébként szerintem jó, ha a test_step ugyanúgy a loss-t adja vissza, mint a training_step.

Probléma 2: Véletlenül maradt így, javítani fogom, köszi!
